'X IO Error' message on systems without a domain name
See IZ #11777
Index: oo_cvs/sal/osl/unx/socket.c
===================================================================
RCS file: /cvs/porting/sal/osl/unx/socket.c,v
retrieving revision 1.17
diff -u -3 -p -r1.17 socket.c
--- oo_cvs/sal/osl/unx/socket.c	9 Apr 2002 18:09:53 -0000	1.17
+++ oo_cvs/sal/osl/unx/socket.c	23 Apr 2003 09:32:44 -0000
@@ -867,7 +867,7 @@ static sal_Bool  _osl_getDomainName (sal
 
 			if (execvp ("/bin/domainname", argv) < 0)
 			{
-				exit(-1);
+				_exit(-1);
 			}
 		}
 		else if (pid > 0)
